app/controllers/accounts_controller.rb

```diff
@@ -56,6 +59,10 @@ def show
 
   private
 
+  def https_enabled?
```

Comment:
I am not sure what this method does that the one in ApplicationController doesn't. The `/health` path is not in this controller so that part will not matter.